airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -50,42 +46,44 @@ class VaultBackend(BaseSecretsBackend, LoggingMixin):
     would be accessible if you provide ``{"connections_path": "connections"}`` and request
     conn_id ``smtp_default``.
 
-    :param connections_path: Specifies the path of the secret to read to get Connections.
-        (default: 'connections')
+    :param connections_path: Specifies the path of the secret to read to get Connections
+        (default: 'connections').
     :type connections_path: str
-    :param variables_path: Specifies the path of the secret to read to get Variables.
-        (default: 'variables')
+    :param variables_path: Specifies the path of the secret to read to get Variables
+        (default: 'variables').
     :type variables_path: str
     :param url: Base URL for the Vault instance being addressed.
     :type url: str
-    :param auth_type: Authentication Type for Vault (one of 'token', 'ldap', 'userpass', 'approle',
-        'github', 'gcp', 'kubernetes'). Default is ``token``.
+    :param auth_type: Authentication Type for Vault. Default is ``token``. Available values are in
+        :py:const:`airflow.providers.hashicorp.hooks.vault.VALID_AUTH_TYPES`.
```

Comment:
Do the possible values get shown in the rendered docs? If not can it please, otherwise we're needing users to source dive which isn't great.

(Perhaps this means turning `VALID_AUTH_TYPES` in to `class AuthTypes(enum.enum)`? I'm not saying I need this to be an enum type, just I'd like the possible values in the rendered docs, and enum might be a way of achieving that.)

Reply:
I replace it with explicit list. It's better this way.

Reply:
Thinking about this enum may be difficult to set via the `backend_kwargs` from the config file.

Reply:
Yep. No enum here is better.